serve.c

```diff
@@ -41,6 +41,7 @@ int main (int argc, char const *argv[]) {
   mg_mgr_init(&mgr, NULL);
   nc = mg_bind(&mgr, s_http_port, ev_handler);
 
+  mg_set_protocol_http_websocket(nc);
```

Comment:
I see, but this line enables websocket, so I ask is your static site served by this repo is using websocket? Because if your site has requests in this protocol(websocket), it might cause mongoose hang up a tcp. So that's this is fixed by adding this call.

By the way, I think we should add an option like `--enable-ws` to enable this feature, websocket is not an usual stack in static site :-)